www/src/pages/themes/ThemesApp.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link, Route, Redirect } from 'react-router-dom';
+import Product from './Product'
+
+class ThemesApp extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <Route path={`${this.props.match.path}/:id`} component={Product} />
+      </div>
```

Comment:
I developed this smaller Apps pattern but is it a good one? How can we make this better?